src/main/java/touro/ConwaysGameOfLife/Grid.java

```diff
@@ -40,55 +37,39 @@ public void makeMove() {
                 }
             }
         }
-        this.board = tempBoard;
+        board = tempBoard;
     }
 
     public int countAliveNeighbors(int x, int y) {
         int aliveNeighborCount = 0;
         for(Direction d : directions) {
             boolean isAlive = checkDirection(d, x, y);      //check neighbor status
-
-
-            if(isAlive){           //if neighbor cell is has true val, increment live neighbors
+            if(isAlive){                                    //if neighbor cell is has true val, increment live neighbors
                 aliveNeighborCount++;
             }
         }
         return aliveNeighborCount;
     }
 
-    public boolean checkStatus(int x, int y) {
-        //make sure in bounds
-        if (x < 0 || x >= width){
-            isAlive = false;
-        }
-        if(y < 0 || y > height){
-            isAlive = false;
-        }
-        return isAlive;
-    }
-
 
     public boolean isAlive(int x, int y){
         return isAlive;
     }
 
     public void setAlive(int x, int y){
-        // isAlive = true;
-        this.board[x][y] = true;
+//        isAlive = true;     //necessary?
+        board[x][y] = true;
     }
 
     /**
      * @return value of cell is 0 - dead
      */
     public void setDead(int x, int y){
-        //isAlive = false;
-        this.board[x][y] = false;
-    }
+         isAlive = false;
```

Comment:
added isAlive = false because otherwise boolean variable stayed true and then the isAlive() method didn't work properly even when a specific cell really was dead and had a value of false